src/swapd/runtime.rs

```diff
@@ -544,6 +554,10 @@ impl Runtime {
                             self.syncer_state.monero_syncer(),
                             Request::SyncerTask(watch_height_xmr_task),
                         )?;
+
+                        // syncer takes too long to give a fee
+                        std::thread::sleep(Duration::from_secs_f32(5.0));
+
```

Comment:
@TheCharlatan, the fee_estimate command takes far too long to return from Syncer to Swapd.

Could we poll and cache values at the level of the syncer, and those are returned directly. i'd say every 20 s should be enough.

Reply:
The sleep there is to avoid the race. 

A lot of the things happen, message goes through the remote peer, and comes back and we still dont have a fee estimate

Reply:
The caching can be done, yes. Nevertheless it would be nice to not rely on the fee estimator to return in time. Do we make similar assumptions for other requests?

Reply:
Ok, I'm reworking the entire fee estimation task for the syncer now.